config/linux-nitropad-x.config

```diff
@@ -1003,7 +1009,7 @@ CONFIG_DMI_SCAN_MACHINE_NON_EFI_FALLBACK=y
 # CONFIG_ISCSI_IBFT is not set
 # CONFIG_FW_CFG_SYSFS is not set
 CONFIG_SYSFB=y
-# CONFIG_SYSFB_SIMPLEFB is not set
+CONFIG_SYSFB_SIMPLEFB=y
```

Comment:
2.4.1 was not pushing this compared to master https://github.com/Nitrokey/heads/blob/50e03c1cfe8ce2cb1d7ad2c43c15d17d05c82dc1/config/linux-nitropad-x.config#L1005